dodola/core.py

```diff
@@ -28,15 +35,26 @@ def bias_correct_bcsd(
         variable name used in training data
     out_variable : str
         variable name used in downscaled output
+    method : str
```

Comment:
Because we only allow certain inputs for the `method` parameter here, I suggest changing this line to
```
    method : {"BCSD", "QDM"}
```

Reply:
good catch, updated.